### DISCUSSION TITLE
在wsl2 中 按照 https://xiaomeng.feishu.cn/wiki/wikcnA2jU2Vfp42lkKJLLuKbKRd 安装1panel 在 这里 [1Panel Log]: 您设置的端口为：遇到这个错误cat: write error: Broken pipe

### DISCUSSION THREAD

**Discussion by jasperjiasc** [1Panel-dev/1Panel#5966] [Q&A]
### 1Panel 版本

1.10.13lts

### 问题描述

在wsl2 中 按照 https://xiaomeng.feishu.cn/wiki/wikcnA2jU2Vfp42lkKJLLuKbKRd 安装1panel 在 这里 [1Panel Log]: 您设置的端口为：遇到这个错误cat: write error: Broken pipe 

### 重现步骤

在wsl2 中 按照 https://xiaomeng.feishu.cn/wiki/wikcnA2jU2Vfp42lkKJLLuKbKRd 安装1panel 在 这里 [1Panel Log]: 您设置的端口为：遇到这个错误cat: write error: Broken pipe

**Comment by wanghe-fit2cloud** [comment]
这个错误可能是由于 `install.sh` 脚本在 WSL2 环境中的适配问题。
建议自行修改脚本来解决此问题。
